auth0/nextjs-auth0
# Docs: passing user from Server Component

- [ ] All new/changed/fixed functionality is covered by tests (or N/A) N/A
- [ ] I have added documentation for all new/changed functionality (or N/A) N/A



### 📋 Changes

Updates the docs for `UserProviderProps` as well as updates `example-app` to demonstrate how to pass a user from a Server Component (as discussed in #1571).



### 📎 References


https://github.com/auth0/nextjs-auth0/issues/1571

### 🎯 Testing


1. Set the required env vars for the example-app
2. Run `npm run start:example`
3. Navigate to http://localhost:3000/
4. Click Login and enter credentials.
5. Upon returning to http://localhost:3000, refresh the page and confirm the "Login" link is not briefly shown.

Closes #1571
